neurom/fst/_neuritefunc.py

```diff
@@ -318,6 +326,30 @@ def section_radial_distances(neurites, neurite_type=NeuriteType.all, origin=None
     return dist
 
 
+def section_term_radial_distances(neurites, neurite_type=NeuriteType.all, origin=None):
+    '''Get the radial distances of the termination sections for a collection of neurites'''
+    dist = []
+    for n in iter_neurites(neurites, filt=is_type(neurite_type)):
+        pos = n.root_node.points[0] if origin is None else origin
+        dist.extend([sectionfunc.section_radial_distance(s, pos)
```

Comment:
extend can take an iterator, so don't need the [ ]

Reply:
OK, will change this.

Reply:
Btw I noticed that all the other functions (such as section_radial_distances) also have the [ ] structure.. Should I keep it for consistency or modify it anyways? 
```
 dist.extend([sectionfunc.section_radial_distance(s, pos)
```

Reply:
they should be fixed, too

Reply:
yup